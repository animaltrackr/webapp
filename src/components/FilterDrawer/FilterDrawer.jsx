import React from 'react';

import Drawer from 'antd/lib/drawer';

import DateRangeForm from '../DateRangeForm/DateRangeForm';

class FilterDrawer extends React.Component {
	render() {
		return (
			<div>
				<Drawer
					title="Filter Results by Time Range"
					placement="right"
					closable={false}
					onClose={this.props.toggleDrawer}
					visible={this.props.drawerVisible}
					width={'350px'}
				>
					<DateRangeForm
						handleDateFilter={this.props.handleDateFilter}
						handleDrawer={this.props.toggleDrawer}
					/>
				</Drawer>
			</div>
		);
	}
}

export default FilterDrawer;
