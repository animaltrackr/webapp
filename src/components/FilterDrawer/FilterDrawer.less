.x-icon {
  color: @text-color-secondary;
  margin-top: 3px;
  font-size: 16px;
  transition: color 0.2s ease;
  float: right;

  &:hover {
    color: @primary-color;
  }
}