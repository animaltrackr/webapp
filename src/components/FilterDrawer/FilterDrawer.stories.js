import React from 'react';
import { storiesOf } from '@storybook/react';
import FilterDrawer from './FilterDrawer';

storiesOf('FilterDrawer', module).add('FilterDrawer', () => (
	<div>
		<FilterDrawer />
	</div>
));
