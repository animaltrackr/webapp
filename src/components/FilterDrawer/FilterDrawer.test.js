import React from 'react';
import { shallow } from 'enzyme';
import FilterDrawer from './FilterDrawer';

describe('<FilterDrawer />', () => {
	test('renders', () => {
		const wrapper = shallow(<FilterDrawer />);
		expect(wrapper).toMatchSnapshot();
	});
});
