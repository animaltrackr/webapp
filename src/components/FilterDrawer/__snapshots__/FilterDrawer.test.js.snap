// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterDrawer /> renders 1`] = `ShallowWrapper {}`;
